x-pack/plugins/ml/public/application/components/data_grid/feature_importance/decision_path_chart.tsx

```diff
@@ -0,0 +1,143 @@
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  AnnotationDomainTypes,
+  Axis,
+  AxisConfig,
+  Chart,
+  LineAnnotation,
+  LineAnnotationDatum,
+  LineSeries,
+  PartialTheme,
+  Position,
+  RecursivePartial,
+  ScaleType,
+  Settings,
+} from '@elastic/charts';
+import { EuiIcon } from '@elastic/eui';
+
+import React, { useCallback, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { DecisionPathPlotData } from './use_classification_path_data';
+
+const baselineStyle = {
+  line: {
+    strokeWidth: 1,
+    stroke: 'gray',
+    opacity: 1,
+  },
+  details: {
+    fontSize: 12,
+    fontFamily: 'Arial',
+    fontStyle: 'bold',
+    fill: 'gray',
+    padding: 0,
+  },
+};
+
+const axes: RecursivePartial<AxisConfig> = {
+  tickLabelStyle: {
+    fontSize: 12,
+  },
+};
+const theme: PartialTheme = {
+  axes,
+};
+
+interface DecisionPathChartProps {
+  decisionPathData: DecisionPathPlotData;
+  predictionFieldName?: string;
+  baseline?: number;
+  minDomain: number | undefined;
+  maxDomain: number | undefined;
+}
+
+const DECISION_PATH_MARGIN = 125;
+const DECISION_PATH_ROW_HEIGHT = 10;
+
+const AnnotationBaselineMarker = <EuiIcon type={'empty'} size={'s'} />;
+
+export const DecisionPathChart = ({
+  decisionPathData,
+  predictionFieldName,
+  minDomain,
+  maxDomain,
+  baseline,
+}: DecisionPathChartProps) => {
+  // adjust the height so it's compact for items with more features
+  const baselineData: LineAnnotationDatum[] = useMemo(
+    () => [
+      {
+        dataValue: baseline ? parseFloat(baseline.toPrecision(3)) : undefined,
+        details: i18n.translate(
+          'xpack.ml.dataframe.analytics.explorationResults.decisionPathBaselineText',
+          {
+            defaultMessage:
+              'baseline (average of predictions for all data points in the training data set)',
+          }
+        ),
+      },
+    ],
+    [baseline]
+  );
+  const tickFormatter = useCallback((d) => Number(d).toPrecision(3), []);
+
+  return (
+    <Chart
+      size={{ height: DECISION_PATH_MARGIN + decisionPathData.length * DECISION_PATH_ROW_HEIGHT }}
+    >
+      <Settings theme={theme} rotation={90} />
+      {baseline && (
+        <LineAnnotation
+          id="xpack.ml.dataframe.analytics.explorationResults.decisionPathBaseline"
+          domainType={AnnotationDomainTypes.YDomain}
+          dataValues={baselineData}
+          style={baselineStyle}
+          marker={AnnotationBaselineMarker}
+        />
+      )}
+
+      <Axis
+        id={'xpack.ml.dataframe.analytics.explorationResults.decisionPathXAxis'}
+        tickFormat={tickFormatter}
+        title={i18n.translate(
+          'xpack.ml.dataframe.analytics.explorationResults.decisionPathXAxisTitle',
+          {
+            defaultMessage: "Prediction for '{predictionFieldName}'",
+            values: { predictionFieldName },
+          }
+        )}
+        showGridLines={true}
```

Comment:
```suggestion
        showGridLines={false}
```

Suggest to hide the vertical grid lines.

Reply:
Updated here https://github.com/elastic/kibana/pull/73561/commits/a71e190ca0f7d9d7edca23ed2b110e5eb6945a7f